test/karma-coverage.config.babel.js

```diff
@@ -30,8 +30,6 @@ module.exports = config => {
       }
     },
 
-    dir: 'dist/test-reports/coverage/',
```

Comment:
We're not actually using it here (none of our reporters actually write anything)